src/components/ListItem/ListItem.tsx

```diff
@@ -181,8 +186,9 @@ const ListItem = (props: ListItemProps) => {
       </div>
       <div className={classes.item}>
         <span className={classes.title}>
-          {title}{account && <b className={classes.appName}>&nbsp;- {account}</b>}
+          {title}{account && <b className={classes.appName}>- {account}</b>}
           {isTeamhub && type === 'TAB' && <Icon icon={Icons.ARROW_UP_LEFT} className={classes.tab}/>}
+          {!isTeamhub && !isRecent && type === 'TAB' && (<span className={classes.open}>{`- open`}</span>)}
```

Comment:
It doesn't work like that?
```suggestion
          {!isTeamhub && !isRecent && type === 'TAB' && (<span className={classes.open}>- open</span>)}
```